CONTRIBUTING.rst

```diff
@@ -14,6 +14,12 @@ Also, please, follow the code style proposed by `PEP8
 <https://www.python.org/dev/peps/pep-0008/>`_. There is a tool called
 pycodestyle, formerly known as pep8, that checks if your code is compliant.
 
+Contributing packages
+---------------------
+
+For details on adding new packages to the project, refer to the versions repository documentation `here
+<https://github.com/open-power-host-os/versions>`_.
```

Comment:
Place a link directly to the contributing page: `https://github.com/open-power-host-os/versions/blob/master/CONTRIBUTING.rst`